src/simulator/PlayerActions.ts

```diff
@@ -0,0 +1,118 @@
+import {ContainmentPolicy, WorldState, SimulatorMetrics, PlayerActions} from '../Simulator/SimulatorModel'
+
+/* 
+    Model lockdowns (source for effects of policies on R -- http://epidemicforecasting.org/containment-calculator)
+    - adjusted for lower real-world effectiveness (86% --> 70%)
+    - perhaps because people are taking this lockdown less seriously (74% of people believe this to be true) source: YouGov 
+
+    Base 'lockdown' models a full lockdown with medium compliance which reduces effectiveness: 
+        
+        Paramaters
+        ----------
+        - Non-essential businesses closed
+        - mask wearing enforced
+        - symptomatic testing enforced
+        - limited gatherings to <10
+        - schools / unis closed
+    
+*/
+
+export const lockdown: ContainmentPolicy = {
+    id: 'lockdown',
+    icon: '',
+    requirements: [],
+    name: 'Lockdown',
+    activeLabel: 're-enforced',
+    inactiveLabel: 'lifted',
+    immediateEffect: (context: WorldState) => ({...context.metrics}),
+    recurringEffect: (context: WorldState) => ({
+        ...context.metrics,
+        r: context.metrics.r - ( context.metrics.r * 0.70 ),  
+    })
+}
```

Comment:
Containment policies just modify R. We can simplify by removing over time effects